HackingWithSwift/week1/day_4/Repeat loops/Repeat loops.playground/Contents.swift

```diff
@@ -0,0 +1,21 @@
+var number = 5
+repeat{
+    print(number)
+    number += 1
+} while number <= 20
+            print("Obema boy, ma big my boss")
+            
+            var pages = 5
+            repeat{
+                print(pages)
+                pages += 1
+            } while pages <= 20
+                        print("Hello")
+            
+
+                        let numbers = [1, 2, 3, 4, 5]
+                        var random: [Int]
+
+                        repeat {
+                            random = numbers.shuffled()
+                        } while random == numbers
```

Comment:
Wow, something went wrong with the formatting of the code :D Please take a look and fix it. (Maybe the `control + i` hotkey will help you). The `repeat` is also used quite rarely, but it's nice that you've familiarized yourself with it.